doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -5,28 +5,65 @@
 
 ## Table of Contents
 * [3.1 Introduction](#3.1)
-* [3.2 Measurement Criteria](#3.2)
-  * [3.2.1 Storage and IOPS](#3.2.1)
-* [3.3 Measurement Types](#3.3)
-  * [3.3.1 Performance Measurements](#3.3.1)
-  * [3.3.2 Resiliency Measurements](#3.3.2)
-* [3.4 NFVI Test Cases](#3.4)
+* [3.2 Assumptions](#3.2)
+* [3.3 Requirement Type](#3.3)
+* [3.4 Profile Catalog](#3.4)
+* [3.5 Software & Hardware Reference](#3.5)
+* [3.6 Options & Extensions](#3.6)
+* [3.7 Measurement Criteria](#3.7)
+  * [3.7.1 Storage and IOPS](#3.7.1)
+* [3.8 Measurement Types](#3.8)
+  * [3.8.1 Performance Measurements](#3.8.1)
+  * [3.8.2 Resiliency Measurements](#3.8.2)
+* [3.9 NFVI Test Cases](#3.9)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
 > Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
 
 <a name="3.2"></a>
-## 3.2 Measurement Criteria
+## 3.2 Assumptions
+
+Content to be written:
+- Assumptions:  Automatable, Integrated with CICD tool chain
+
+<a name="3.3"></a>
+## 3.3 Requirement Type
+
+Content to be written:
+-  Type of requirement:  Bare metal, API, etc
+
+<a name="3.4"></a>
+## 3.4 Profile Catalog
+
+Content to be written:
+-  Table showing Profile Catalog
```

Comment:
It is not clear to me what this table should show: which parts of the profile catalogue is covered by the LFN compliance program?

Reply:
@gkunz - This is placeholder, or table, to describe the differences between the different profiles which we would need to test.  In other words, Basic(B), Network(N), and Compute(C) intensive profiles.  For compliance, verification, and certification, of NFVI solutions provided for a given NFVI Profile, it is required to have a reference implementation of each profile so it can be used for compliance, validation, and certification.  Those reference implementations need to reflect on their corresponding profiles and deliver all metrics and capabilities promised.   Please refer to a prior document we wrote which we can use/pull-in, as needed (called [RM Ch 8](https://github.com/cntt-n/CNTT/blob/d275de2153425215e4c44becffb1b342d2a0f760/doc/ref_model/chapters/chapter08.md)).

Reply:
I don't understand why this table is supposed to describe the differences between the profiles. To my understanding, the profiles are described in the RM and the RAs, including the capabilities of those profiles. For the compliance program (this chapter), we have to select which of those profiles and which capabilities (e.g., performance results, etc.) are part of the scope.

Reply:
Agree.  Having a reference table here, even is abbreviated with a hyperlink back to the more detailed RM/RA requirements docs will serve as explicit reference to the profile under test.  This section/chapter provides an explicit artifact and evidence as to what is being implemented and certified.